### PR TITLE
[FSET-1619] Create new FS Assessor account and add skills

### PR DESCRIPTION
**Pull request by mgilgar**
4 PRs:
fset-faststream-admin-frontend
https://github.com/hmrc/fset-faststream/pull/363
csr-auth-provider
https://github.com/hmrc/fset-email-renderer/pull/15

Bear in mind it will be a second version soon.

### CODE REVIEW

File: app/controllers/AssessorController.scala [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+}
+
+trait AssessorController extends BaseController {
+
+  val assessorService: AssessorService
+
+
+  def saveAssessor(userId: String): Action[JsValue] = Action.async(parse.json) { implicit request =>
+    withJsonBody[Assessor] { assessor => assessorService.saveAssessor(userId, assessor).map (_ =>
+      Ok(s"I was called with userId: $userId")
```
Comment (kjharkin) [inline app/controllers/AssessorController.scala]: What is this text? Please remove

File: app/controllers/AssessorController.scala [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+    )}
+  }
+
+  def addAvailability(userId: String): Action[JsValue] = Action.async(parse.json) { implicit request =>
+    withJsonBody[AssessorAvailability] { availability => assessorService.addAvailability(userId, availability).map (_ =>
+        Ok(s"I was called with userId: $userId")
```
Comment (kjharkin) [inline app/controllers/AssessorController.scala]: What is this text? Please remove.

File: conf/app.routes [hunk 3]
```diff
@@ -114,9 +114,12 @@ DELETE      /application/:applicationId/issue                            control
 
 GET         /schools                                                     controllers.SchoolsController.getSchools(term: String)
 
-POST        /assessor-availability/save/:userId                          controllers.AssessorAvailabilityController.save(userId: String)
-GET         /assessor-availability/find/:userId                          controllers.AssessorAvailabilityController.find(userId: String)
-GET         /assessor-availability/countSubmitted                        controllers.AssessorAvailabilityController.countSubmitted()
+POST        /assessor/save/:userId                                       controllers.AssessorController.saveAssessor(userId: String)
+GET         /assessor/find/:userId                                       controllers.AssessorController.findAssessor(userId: String)
+# TODO: /assessor-availability rename to sth.
```
Comment (kjharkin) [inline conf/app.routes]: TODOs should only be used for your own purposes and have to be removed before code is merged.

Comment (mgilgar) [inline conf/app.routes]: We remove it but how can we communicate to the person who will do availability that this should be changed?

File: app/controllers/AssessorController.scala (hunk 1 above)
Comment (mgilgar) [inline app/controllers/AssessorController.scala]: will change

File: app/controllers/AssessorController.scala (hunk 2 above)
Comment (mgilgar) [inline app/controllers/AssessorController.scala]: will change

File: conf/app.routes (hunk 3 above)
Comment (kjharkin) [inline conf/app.routes]: Certainly not through leaving instructions in the code - either it should be refactored as part of your ticket (if it results in 'wrong' code) or it is not required.

**Pull request merged**